include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -1158,19 +1167,19 @@ class charge_distribution_surface<Lyt, false> : public Lyt
      * @param history_mode charge_distribution_history::NEGLECT if the information (local electrostatic energy) of the
      * previous charge distribution is used to make the update more efficient, charge_distribution_history::CONSIDER
      * otherwise.
```

Comment:
```suggestion
     * @param history_mode `charge_distribution_history::NEGLECT` if the information (local electrostatic energy) of the
     * previous charge distribution is used to make the update more efficient, `charge_distribution_history::CONSIDER`
     * otherwise.
```